baseapp/msg_service_router.go

```diff
@@ -40,6 +40,10 @@ func NewMsgServiceRouter() *MsgServiceRouter {
 }
 
 func (msr *MsgServiceRouter) SetCircuit(cb CircuitBreaker) {
+	if cb == nil {
```

Comment:
why do we need to check here if we check in another place: https://github.com/cosmos/cosmos-sdk/blob/96b8e33f1471b7d6d51b48ee796cb836feee28e0/baseapp/msg_service_router.go#L137 this seems redundant